osu.Game.Rulesets.Catch/Skinning/Default/FadePiece.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Rulesets.Catch.Skinning.Default
+{
+    public class FadePiece : CircularContainer
```

Comment:
is there a reason this (and `GlowPiece`) got extracted from the `DefaultCatchHitExplosion`?

Reply:
I was using osu!standard's hit lighting as a reference and it had all the parts of the hit explosion had their own types. See mainly `osu.Game.Rulesets.Osu/Skinning/Default/FlashPiece.cs`. 

This is just why I did that, and I can just put it back to how it was considering that as of right now both types are just `CircularContainer`s with defaults, rather that implementing any functionality.

Reply:
I'm not sure splitting was necessary. These classes are so small I'd just put them back where they were (not to mention that they aren't really properly written - if you wanted to split this to a component, the base class should be `CompositeDrawable` and the `CircularContainer` should be the `InternalChild` here).